lib/safer_rails_console/patches/sandbox/auto_rollback.rb

```diff
@@ -35,10 +47,24 @@ def execute_and_clear(...)
         end
 
         if defined?(::ActiveRecord::ConnectionAdapters::PostgreSQLAdapter)
-          ::ActiveRecord::ConnectionAdapters::PostgreSQLAdapter.prepend(PostgreSQLAdapterPatch)
+          if SaferRailsConsole::RailsVersion.eight_or_above?
+            ::ActiveRecord::ConnectionAdapters::PostgreSQLAdapter.prepend(PostgreSQLAdapteRails8Patch)
+          elsif SaferRailsConsole::RailsVersion.six_or_above?
+            ::ActiveRecord::ConnectionAdapters::PostgreSQLAdapter.prepend(PostgreSQLAdapteRails67Patch)
+          end
```

Comment:
I'm worried that `8Patch` won't make sense when we have Rails 9+, so what do you think about calling the 8+ patch, `PostgreSQLAdapterPatch` and the old one, `LegacyPostgreSQLAdapterPatch`?

Reply:
👍  Good idea, done!